subprojects/docs/src/docs/release/notes.md

```diff
@@ -319,6 +319,23 @@ dependencyResolutionManagement {
 
 You can learn more about declaring rules globally in the [user manual](userguide/component_metadata_rules.html#sec:rules_in_settings).
 
+### Locking of Gradle settings dependencies
+
+[Dependency locking](userguide/dependency_locking.html) makes builds using dynamic versions deterministic.
+
+So far you could lock dependency configurations from your project or from the project buildscript classpath.
+This release adds support to lock configurations from the Gradle `settings.gradle(.kts)`:
```

Comment:
I'd also add a link to https://docs.gradle.org/current/userguide/build_lifecycle.html#sec:settings_file